best-practices/tidb-best-practices.md

```diff
@@ -1,199 +1,200 @@
----

```

Comment:
The diff view is weird. As I confirmed, the only difference I made was adding a line of aliases, but the diff view is not displayed correctly. This happens for several large files in this PR. FYI.